porch/api/porch/types_packagerevisionresources.go

```diff
@@ -45,6 +45,9 @@ type PackageRevisionResourcesSpec struct {
 	// PackageName identifies the package in the repository.
 	PackageName string `json:"packageName,omitempty"`
 
+	// Description identifies the description of the package.
+	Description string `json:"description,omitempty"`
```

Comment:
I agree that `description` sounds like some free text, so we should change the name. I think at first glance `revisionName` makes sense, but it might be confusing when you see both the `revision` and `revisionName` fields. It can get even more confusing when you consider that the object is called a `PackageRevision` and has its own `metadata.name` and `packageName`.  I feel like the words `revision` and `name` are becoming overloaded.

WDYT about calling it `tag`?

Reply:
Changed to `workspaceName`.